Makefile

```diff
@@ -12,11 +12,19 @@ export GO111MODULE = on
 ifeq ($(OS),Windows_NT)
        # Just assume we're in interactive mode on Windows
        INTERACTIVE = 1
-       VERSION = $(shell type version.txt)
+			 VERSION = $(shell type version.txt)
 else
        INTERACTIVE := $(shell [ -t 0 ] && echo 1)
 endif
 
+ifdef GO_SPACEMESH_VERSION
+	VERSION=$(GO_SPACEMESH_VERSION)
+endif
+
+ifdef WINDOWS_BUILD_DIR
+	BIN_DIR_WIN=$(WINDOWS_BUILD_DIR)
```

Comment:
I think you can avoid this clause entirely by changing line 8 to:

`BIN_DIR_WIN ?= $(CURR_DIR_WIN)/build`

(and passing this value in as `BIN_DIR_WIN` instead of `WINDOWS_BUILD_DIR`)